avocado/test.py

```diff
@@ -11,7 +11,6 @@
 # This code was inspired in the autotest project,
 # client/shared/test.py
 # Authors: Martin J Bligh <mbligh@google.com>, Andy Whitcroft <apw@shadowen.org>
-
```

Comment:
sorry for being pedantic: I would keep this space here, usually (in our code) there's a space between the comments and the module's docstring that follows.